pyproject.toml

```diff
@@ -37,7 +37,7 @@ dependencies = [
     "scipy>=1.4.0,<2.0.0",
     "h5py>=3.1.0,<4.0.0",
     "opencv-python>=4.5.0,<5.0.0",
-    "pypdfium2>=2.1.0,<3.0.0",  # cf. https://github.com/mindee/doctr/issues/947
+    "pypdfium2>=3.0.0,<4.0.0",
```

Comment:
Maybe we can pin:
```
pypdfium2~=3.1.0
```

What do you think @felixdittrich92 ? Do you prefer less constraints in `pyproject` as it has been done with other dependencies ?

Reply:
@odulcy-mindee i think this is fine as it is :) but to be sure :
@mara004 in future releases with breaking changes you increase the major version number correct ?

Reply:
the only thing we want to avoid are breaking changes which leads to issues in doctr so keeping a bigger range of minor and patch updates should be ok :)

Reply:
Yes. Maybe I might change some really insignificant parameter without a major update if there's a problem with it, but document opening and rendering methods will remain the same within one major version.

Reply:
Okey, it's fine for me, thanks for the quick reply @mara004 !